BoxContentSDK/BoxContentSDKTests/BOXFolderTests.m

```diff
@@ -100,6 +100,18 @@ - (void)test_that_folder_with_default_fields_is_parsed_correctly_from_json
     XCTAssertEqual(folder.metadata.count, 0);
 }
 
+- (void)test_that_folder_with_missing_default_invitee_role_is_parsed_correctly_from_json
+{
```

Comment:
could we do the same for files? same for below

Reply:
The reason I didn't do this for files is because the unpacking logic itself doesn't distinguish between files & folders.  

I'm happy to add that coverage if we think it's possible that in the future the ode-under test might fork based on is-file/is-folder. If that isn't likely... then the added coverage wouldn't really be adding much value.